vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -509,7 +509,7 @@ class MessageItemFactory @Inject constructor(
                 allowNonMxcUrls = informationData.sendState.isSending()
         )
 
-        val playable = messageContent.mimeType == MimeTypes.Gif
+        val playable = messageContent.mimeType == MimeTypes.Gif || messageContent.mimeType == MimeTypes.Webp
```

Comment:
This is a relatively poor check, but it's the best we can do without being able to read the file headers. 

In theory, we could also use 
```suggestion
        val playable = messageContent.mimeType == MimeTypes.Gif || (messageContent.mimeType == MimeTypes.Webp && Build.VERSION.SDK_INT >= Build.VERSION_CODES.P)
```
to not show the "play" button on older versions of Android, but it doesn't really make much of an appreciable difference - they're broken either way, and were before.

Reply:
I don't have any statistics, but I don't think animated webps are commonly used compared to static webp images, so making all webp images 'playable' might not be the best approach. 

The `playable` attribute is only used to display the 'play' button above the image, but if you have the autoplay animated images option enabled the image will already be playing in the timeline, so given these 2 possible cases:

1. Users not seeing the play button in an animated webp image and not having autoplay images enabled, thus not knowing that it's an animated image.
2. Users seeing play buttons over every single webp, most of them being static.

I think it would be better to go with the first option, that is: remove the `|| messageContent.mimeType == MimeTypes.Webp`.

Reply:
Sadly, autoplay in thumbnail doesn't work unless `playable` is enabled.

https://github.com/vector-im/element-android/blob/de50577ac38ebd9f2b69715142de1520bf9299eb/vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt#L552

It does work when tapping into it, which is still a better result than it not working at all. 
If we want to actually have it autoplay in preview mode, we'd need to know beforehand that it's actually an animated webp.

Reply:
I'm actually seeing the opposite issue. `playable` is disabled, autoplay is disabled too, but thumbnails for animated webp images are autoplayed anyway. I don't think it's directly related to your changes, but I also have no idea why in your case it seems like the opposite is happening.

<details><summary>See recording</summary>
Top and bottom items are WebP, middle is a gif.

[Play recoding](https://user-images.githubusercontent.com/480955/220705383-2e8a13a1-7478-4ba2-8f31-0e01beb6d57e.webm)
</details>

Reply:
I've been able to replicate both behaviors now. I'm going to do some testing to figure out what causes it to autoplay _sometimes_ when autoplay is turned off. In your recording, you can see that the nyancat gif has squared corners, and I'm currently looking at my emulator on API 33 and it has rounded corners and doesn't play until I click into it, even if I tap in and then go back out, and if I turn autoplay on, go back, then go turn it off again.

However, my laptop is now showing the same behavior as yours is.

I'll let you know when I have an explanation for this. Thanks for spotting!